workflow-service/src/main/java/com/redhat/parodos/security/LocalSecurityConfiguration.java

```diff
@@ -36,43 +38,15 @@
 
 @Profile("local")
```

Comment:
We changed profile to `dev` [here](https://github.com/redhat-developer/parodos/pull/82/files#diff-a547c06eb8a140e1fc0f5e3b1d977f0055bf3ed5b09158296b24ff775e1fa878R39). Should this be updated as well?

Reply:
It's ok, mainly because local uses the internal ldif server and dev uses the ldap connection to the openldap server.